src/pages/invoice.js

```diff
@@ -371,18 +373,29 @@ export class InvoicePage extends React.Component {
                       <Container
                         display="flex"
                         justifyContent="space-between"
+                        flexBasis="100%"
                         px={3}
                         py={2}
                         background="#ebf4ff"
                         fontWeight="bold"
                       >
                         <FormattedMessage id="total" defaultMessage="TOTAL" />
                         <Span>{formatCurrency(invoice.totalAmount, invoice.currency)}</Span>
+                        
                       </Container>
+                      
                     </Container>
+
+                    
                   </Flex>
                 )}
+                
+               
               </Box>
+              <P mt={2} textAlign="left" color="black" padding="4rem 0">
+                  {invoice.extraInfo || 
+                  <FormattedMessage id="invoice.donationAppreciation" defaultMessage="" />}
```

Comment:
I don't understand the purpose of this `FormattedMessage`?

Reply:
i was your using it to display a message when it was not connected to the api ill take it out now